### DISCUSSION TITLE
FluxStore - support osclass

### DISCUSSION THREAD

**Discussion by BongToeun** [inspireui/support#5125] [Request new Native Payment]
Hi Flux developers and osclass user
Please kindly help add support osclass.
Osclass.osclasspoint.com
Alot of user use osclass for build classified website for their business, osclass already have API plugin from osclass market.  Osclasspoint.com
Osclass don't have mobile app yet, i think it's the best time for you to build first mobile for them, 
The classified UI like your woocommerce app or your  other store UI.

Hope you think about it and develop it for us.
Thank you for your intergrade support on osclass
Regard

**Comment by BongToeun** [comment]
Please add support for osclass

**Comment by techibaba** [comment]
Add a app for osclass also
